src/OpenTelemetry.Extensions.AzureMonitor/ApplicationInsightsSampler.cs

```diff
@@ -38,6 +41,14 @@ public ApplicationInsightsSampler(float samplingRatio)
 
         this.samplingRatio = samplingRatio;
         this.Description = "ApplicationInsightsSampler{" + samplingRatio + "}";
+        var sampleRate = (int)Math.Round(samplingRatio * 100);
```

Comment:
Not added by this PR, but is there any particular reason to prefer the type `float` over `double` for `samplingRatio`?

Reply:
It's based on Application Insights Sampler Specification.